gstd/gstd.c

```diff
@@ -200,12 +107,8 @@ main (gint argc, gchar * argv[])
   g_option_context_add_main_entries (context, entries, NULL);
 
   /* Initialize GStreamer */
-  gstreamer_group = gst_init_get_option_group ();
-  g_option_context_add_group (context, gstreamer_group);
-
-  /* Read option group for each IPC */
-  ipc_add_option_groups (ipc_array, supported_ipcs, num_ipcs, context,
-      optiongroup_array);
+  gstd_manager_new (&manager, 0, NULL);
+  gstd_get_option_context (manager, &context);
```

Comment:
In this case, the name is not appropriate. We are not getting any context. We are appending our groups to the context. Please rename accordingly.

Reply:
Got it. `gstd_context_add_group` was used instead